src/main/java/at/spengergasse/cooking/recipes/service/image/ImageService.java

```diff
@@ -0,0 +1,49 @@
+package at.spengergasse.cooking.recipes.service.image;
+
+import io.minio.MinioClient;
+import io.minio.PutObjectArgs;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.InputStream;
+import java.time.Instant;
+
+@AllArgsConstructor
+@Service
+public class ImageService {
+
+    final private MinioClient minioClient =
+            MinioClient.builder()
+                    .endpoint("http://minio:9000")
+                    .credentials("minioadmin", "devPassword") //TODO: not use root credentials, would need to be generated in docker compose somehow so that it works for everyone
+                    .build();
+
+    public String uploadImage(MultipartFile image){
+        try {
+            // Generate a unique object name based on the current timestamp
+            String objectName = Instant.now().toEpochMilli() + "_" + image.getOriginalFilename();
```

Comment:
understand the concept, not sure if I like it ...
explain your arguments for prefixing ... is kind a hiding/pushing back the important information